corehq/apps/domain/forms.py

```diff
@@ -115,7 +115,7 @@
 from corehq.apps.users.models import CouchUser, WebUser
 from corehq.apps.users.permissions import can_manage_releases
 from corehq.toggles import HIPAA_COMPLIANCE_CHECKBOX, MOBILE_UCR, \
-    SECURE_SESSION_TIMEOUT, MONTIOR_2FA_CHANGES
+    SECURE_SESSION_TIMEOUT, MONITOR_2FA_CHANGES
```

Comment:
I tried googling `Montior` to see if it was a name for a town. Instead, I found articles of other people in the same predicament. alas, I'm not alone.

Reply:
I'd like us all to imagine the sun setting on Lake Montiór located on an earthlike exoplanet in a galaxy far, far away
![lake montior planet](https://user-images.githubusercontent.com/716573/91362271-c0f0d580-e7bf-11ea-9c88-a03b979bda16.jpg)